WSFrontend/org.openl.rules.ruleservice.deployer/src/org/openl/rules/ruleservice/deployer/RulesDeployerService.java

```diff
@@ -252,7 +261,7 @@ private void deployInternal(Path pathToArchive,
                     .map(root::resolve)
                     .filter(Files::exists)
                     .filter(Files::isRegularFile)
-                    .map(RulesDeployerService::getDeploymentName)
+                    .map(this::getDeploymentName)

```

Comment:
What is the benefit of using non-static reference? Does It reduce memory consumption or improves performance?

Reply:
getDeploymentName uses a private field with YamlMapper now